src/IB/IBFEMethod.cpp

```diff
@@ -1772,8 +1789,9 @@ IBFEMethod::doInitializeFEEquationSystems()
         // object we only have to do this assignment once
         d_active_fe_data_managers[part]->COORDINATES_SYSTEM_NAME = COORDS_SYSTEM_NAME;
     }
+
     return;
-}
+} // doInitializeFEEquationSystems
```

Comment:
Can we rip all of these out? 😀

(Of course I put most of them there in the first place.)

Reply:
I actually kinda like these - it makes it easier, when you are at the end of some massive function, to see where you are.

If you don't like it then we can argue about it in #1124 :)

Reply:
Yeah, that is the intention!

What I don't like about them is that they are maintained manually.